Plugin not loading in Traefik 2.5

### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.containo.us) and didn't find any.

### What did you do?

Started Traefik 2.5.2 using this plugin ['azurejwttokenvalidation'](https://github.com/dkijkuit/azurejwttokenvalidation) and it fails immediately while it works fine in Traefik 2.4.11 or lower. Could be related to an updated Yaegi version?

### What did you see instead?

Error message:
```
traefik.go:81: command traefik error: github.com/dkijkuit/azurejwttokenvalidation: failed to import plugin code "github.com/dkijkuit/azurejwttokenvalidation": 1:21: import "github.com/dkijkuit/azurejwttokenvalidation" error: plugins-local\src\github.com\dkijkuit\azurejwttokenvalidation\validatetoken.go:226:70: cannot use type [32]uint8 as type []uint8
```

### What version of Traefik are you using?

Traefik version 2.5.2 built on 2021-09-02T15:07:43Z

### What is your environment & configuration?

```yaml
# Static configuration
api:
  dashboard: true
  insecure: true

pilot:
  token: <TOKEN>

log:
  level: DEBUG

experimental:
  localPlugins:
    azure-jwt-check:
      moduleName: github.com/dkijkuit/azurejwttokenvalidation

entryPoints:
  http:
    address: ":4000"
    forwardedHeaders:
      insecure: true

providers:
  file:
    filename: plugins-local/src/github.com/dkijkuit/azurejwttokenvalidation/dynamic-dev-config.yml
```



### If applicable, please paste the log output in DEBUG level
```
time="2021-09-15T14:44:33+02:00" level=info msg="Configuration loaded from file: C:\\Development\\Tools\\Traefik\\plugins-local\\src\\github.com\\dkijkuit\\azurejwttokenvalidation\\static-dev-config.yml"
time="2021-09-15T14:44:33+02:00" level=info msg="Traefik version 2.5.2 built on 2021-09-02T15:07:43Z"
time="2021-09-15T14:44:33+02:00" level=debug msg="Static configuration loaded {\"global\":{\"checkNewVersion\":true,\"sendAnonymousUsage\":true},\"serversTransport\":{\"maxIdleConnsPerHost\":200},\"entryPoints\":{\"http\":{\"address\":\":4000\",\"transport\":{\"lifeCycle\":{\"graceTimeOut\":\"10s\"},\"respondingTimeouts\":{\"idleTimeout\":\"3m0s\"}},\"forwardedHeaders\":{\"insecure\":true},\"http\":{},\"udp\":{\"timeout\":\"3s\"}},\"traefik\":{\"address\":\":8080\",\"transport\":{\"lifeCycle\":{\"graceTimeOut\":\"10s\"},\"respondingTimeouts\":{\"idleTimeout\":\"3m0s\"}},\"forwardedHeaders\":{},\"http\":{},\"udp\":{\"timeout\":\"3s\"}}},\"providers\":{\"providersThrottleDuration\":\"2s\",\"file\":{\"watch\":true,\"filename\":\"plugins-local/src/github.com/dkijkuit/azurejwttokenvalidation/dynamic-dev-config.yml\"}},\"api\":{\"insecure\":true,\"dashboard\":true},\"log\":{\"level\":\"DEBUG\",\"format\":\"common\"},\"pilot\":{\"token\":\"39492e56-a258-4156-940b-bb06e2e79755\",\"dashboard\":true},\"experimental\":{\"localPlugins\":{\"azure-jwt-check\":{\"moduleName\":\"github.com/dkijkuit/azurejwttokenvalidation\"}}}}"
time="2021-09-15T14:44:33+02:00" level=info msg="Stats collection is enabled."
time="2021-09-15T14:44:33+02:00" level=info msg="Many thanks for contributing to Traefik's improvement by allowing us to receive anonymous information from your configuration."
time="2021-09-15T14:44:33+02:00" level=info msg="Help us improve Traefik by leaving this feature on :)"
time="2021-09-15T14:44:33+02:00" level=info msg="More details on: https://doc.traefik.io/traefik/contributing/data-collection/"
2021/09/15 14:44:33 traefik.go:81: command traefik error: github.com/dkijkuit/azurejwttokenvalidation: failed to import plugin code "github.com/dkijkuit/azurejwttokenvalidation": 1:21: import "github.com/dkijkuit/azurejwttokenvalidation" error: plugins-local\src\github.com\dkijkuit\azurejwttokenvalidation\validatetoken.go:227:70: cannot use type [32]uint8 as type []uint8
```

--- a/go.mod
+++ b/go.mod
@@ -62,7 +62,7 @@ require (
 	github.com/stvp/go-udp-testing v0.0.0-20191102171040-06b61409b154
 	github.com/tinylib/msgp v1.0.2 // indirect
 	github.com/traefik/paerser v0.1.4
-	github.com/traefik/yaegi v0.11.1
+	github.com/traefik/yaegi v0.11.2
 	github.com/uber/jaeger-client-go v2.29.1+incompatible
 	github.com/uber/jaeger-lib v2.2.0+incompatible
 	github.com/unrolled/render v1.0.2

